Form-Controls/index.html

```diff
@@ -10,18 +10,57 @@
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1 class="header">Amir T-Shirt Gallery</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>T-Shirt Order Form</title>
+   
+    </head>
+    <body class="form">
+        <h1 class="title">Order Your T-Shirt</h1>
+        <form id="tshirtForm">
+            <label for="customerName">What is your name?</label>
+            <input type="text" id="customerName" placeholder="Enter your name" required minlength="2">
+            
+            <label for="customerEmail">What is your email?</label>
+            <input type="email" id="customerEmail" placeholder="Enter your email" required>
+            
+            <fieldset>
+                <legend>What colour should this t-shirt be?</legend>
+                <label><input type="radio" name="tshirtColor" value="Red" required> Red</label>
+                <label><input type="radio" name="tshirtColor" value="Green"> Green</label>
+                <label><input type="radio" name="tshirtColor" value="Blue"> Blue</label>
+            </fieldset>
+            
+            <label for="tshirtSize">What size do you want?</label>
+            <select id="tshirtSize" required>
+                <option value="" disabled selected>Select your size</option>
+                <option value="XS">XS</option>
+                <option value="S">S</option>
+                <option value="M">M</option>
+                <option value="L">L</option>
+                <option value="XL">XL</option>
+                <option value="XXL">XXL</option>
+            </select>
+            
+            <label for="deliveryDate">When do you want the t-shirt to be delivered?</label>
+            <input type="date" id="deliveryDate" required>
```

Comment:
does this input meet all the requirements for this field?

Reply:
Thanks. I fixed it.